docs/pages/database-access/guides/clickhouse-self-hosted.mdx

```diff
@@ -0,0 +1,305 @@
+
+By default, the created user may not have access to anything and won't be able to connect, so let's grant it some permissions:
+
+```sql
+GRANT ALL ON *.* TO alice;
+```
+
+## Step 4/5. Configure and start the Database Service
+
+Install and configure Teleport on the host or Kubernetes cluster where you will
+run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Server">
+
+(!docs/pages/includes/install-linux.mdx!)
+
+On the host where you will run the Teleport Database Service, start Teleport
+with the appropriate configuration.
+
+Note that a single Teleport process can run multiple different services, for
+example multiple Database Service agents as well as the SSH Service or Application
+Service. The step below will overwrite an existing configuration file, so if
+you're running multiple services add `--output=stdout` to print the config in
+your terminal, and manually adjust `/etc/teleport.yaml`.
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+<Tabs>
+<TabItem label="HTTP">
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=<Var name="teleport.example.com" description="Your Teleport Proxy Service or Teleport Cloud tenant address"/> \
+   --name=example-clickhouse \
+   --protocol=clickhouse-http \
+   --uri=clickhouse.example.com:8443 \
+   --labels=env=dev
+```
+
+</TabItem>
+<TabItem label="Native TCP">
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=<Var name="teleport.example.com" description="Your Teleport Proxy Service or Teleport Cloud tenant address"/> \
+   --name=example-clickhouse \
+   --protocol=clickhouse \
+   --uri=clickhouse.example.com:9440 \
+   --labels=env=dev
+```
+</TabItem>
+</Tabs>
+
+Configure the Database Service to start automatically when the host boots up by
+creating a systemd service for it. The instructions depend on how you installed
+the Database Service.
+
+<Tabs>
+<TabItem label="Package Manager">
+
+On the host where you will run the Teleport Database Service, start Teleport:
+
+```code
+$ sudo systemctl enable teleport
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+<TabItem label="TAR Archive">
+
+On the host where you will run the Teleport Database Service, create a systemd
+service configuration for Teleport, enable the Teleport service, and start
+Teleport:
+
+```code
+$ sudo teleport install systemd -o /etc/systemd/system/teleport.service
+$ sudo systemctl enable teleport
+$ sudo systemctl start teleport
+```
+
+</TabItem>
+</Tabs>
+
+
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+  Teleport provides Helm charts for installing the Teleport Database Service in Kubernetes Clusters.
+
+  (!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+  (!docs/pages/includes/database-access/db-helm-install.mdx dbName="example-clickhouse-http" dbProtocol="clickhouse-http" databaseAddress="clickhouse.example.com:8443" dbName="clickhouse-http" !)
```

Comment:
Does it matter that the Linux Server version provides two options for http and tcp-client, and with Kubernetes there's only the http version?

Reply:
Fixed in [49cc5b5](https://github.com/gravitational/teleport/pull/28460/commits/49cc5b538b300d19b8282d1063c825e604cc89ed)